content/learn/contribute/reference/triage.md

```diff
@@ -121,6 +121,40 @@ There are several paths for PRs to be closed:
 
 When closing a PR, check if it has an issue linked. If it does not, you should strongly consider creating an issue and linking the now-closed PR to help make sure the previous work can be discovered and credited.
 
+## Milestones, Roadmaps and Work Planning
+
+Bevy is, to a large degree, a project powered by volunteers, each with their own motivations and goals.
+The usual estimation and long-term planning difficulties of software are amplified dramatically: this is why Bevy does not maintain a central roadmap.
+
+Despite this, some degree of work planning is important, as both a coordination and quality control mechanism.
+This is generally fairly decentralized and organic, relying on temporary, focused working groups and milestones.
+
+Each release (both major and minor) has an [associated milestone](https://github.com/bevyengine/bevy/milestones).
```

Comment:
Interestingly, we have more milestones than versions. For example, there probably won't be a 0.16.2 release even though we have the milestone. Don't think we need to specify this explicitly, just pointing it out :)
I always assumed that 0.17 just cherry-picks 0.16.2 as well.

Reply:
Yeah, PRs are always merged to main, so there's no cherrypicking needed.